src/views/home/components/bridge-form/bridge-form.view.tsx

```diff
@@ -332,9 +300,11 @@ const BridgeForm: FC<BridgeFormProps> = ({
           </div>
           <div className={classes.rightBox}>
             <Typography type="body2">Balance</Typography>
-            <Typography type="body1">
-              {`${formatTokenAmount(balanceFrom || BigNumber.from(0), token)} ${token.symbol}`}
-            </Typography>
+            <TokenBalance
+              token={{ ...token, balance: balanceFrom }}
```

Comment:
Instead of declaring `balanceFrom` and building here the token, we could also set the token with balance once we get the balances in the effect so the selected token already has the balance.

```diff
-         .then(setTokens)
+        .then((tokensWithBalance) => {
+          setTokens(tokensWithBalance);
+          setToken((selectedToken) =>
+            tokensWithBalance.find((tkn) => tkn.address === selectedToken?.address)
+          );
+        })
```

Reply:
This comment has been outdated by [this one](https://github.com/0xPolygonHermez/zkevm-bridge-ui/pull/231#discussion_r1012870831).